### PR TITLE
Remove outdated TODOs in README.md

### PR DESCRIPTION
**Pull request by terrytangyuan**
I believe these are outdated. TODOs can be found in issues/code.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/katib/468)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -372,9 +371,3 @@ ks delete ${KF_ENV} -c ambassador
 ## CONTRIBUTING
 
 Please feel free to test the system! [developer-guide.md](./docs/developer-guide.md) is a good starting point for developers.
-
-## TODOs
-
-* Integrate KubeFlow (TensorFlow, Caffe2 and PyTorch operators)
-* Support Early Stopping
-* Enrich the GUI
```
Comment (hougangliu) [inline README.md]: these TODOs  aren't still done

Comment (terrytangyuan) [reply README.md]: This is README though. Should they be tracked somewhere else, e.g. roadmap, issues, story board, etc.?

Comment (hougangliu) [reply README.md]: OK, maybe you can create a doc to trace it

Comment (gaocegege) [reply README.md]: I think we could use issues to keep track of the TODO items. WDYT

Comment (hougangliu) [reply README.md]: make sense

**Comment by hougangliu** [comment]
/lgtm
@terrytangyuan Thanks for your first PR of katib.

**Comment by terrytangyuan** [comment]
Anyone wants to merge this tiny PR? :-)

**Pull request merged**